utils/constants/common-constants.ts

```diff
@@ -0,0 +1,33 @@
+import { HomeIcon, BookIcon, SettingIcon } from '../../assets/icons';
+
+interface SidebarItem {
+  id: number;
+  iconName: string;
+  path: string;
+  icon: () => JSX.Element;
+  position: number;
+}
+
+export const sidebarItems: SidebarItem[] = [
```

Comment:
This type of constants we should name it in uppercase, so we can rename this constant as `SIDEBAR_ITEMS` instead of `sidebarItems`.